infrastructure/outputs.tf

```diff
@@ -29,3 +29,19 @@ output "OAUTH2_CLIENT_ID" {
 output "OAUTH2_REDIRECT_URI" {
   value = "${local.oauth2_redirect_uri}"
 }
+
+output "ELASTIC_SEARCH_ENABLED" {
+  value = "${var.elastic_search_enabled}"
+}
+
+output "ELASTIC_SEARCH_HOST" {
```

Comment:
do we need the port and the scheme in data store? At the moment they are passed here
`http://%s:9200"`
Unless we change the line above, having a var for scheme and port might be misleading